privaterelay/templates/includes/login.html

```diff
@@ -49,6 +49,28 @@
 						<img class="" alt="" src="{% static 'images/settings.svg' %}" />
 						{% ftlmsg 'nav-profile-settings' %}
 					</a>
+					{% if user_profile.has_premium %}
+						<a
+							href="{{ settings.FXA_SUPPORT_URL }}?utm_source={{ settings.SITE_ORIGIN }}"
```

Comment:
What is the benefit of tracking the `utm_source` here?

Reply:
`rel="noopener noreferrer"` means SUMO won't be able to see in the referrer that this traffic comes from `relay.firefox.com` unless we explicitly send it in the `utm_source` param.